exercises/1-the-manual-menace/README.md

```diff
@@ -514,29 +514,17 @@ git commit -m "Adding Jenkins and Jenkins s2i"
 git push
 ```
 
-11.  In order for Jenkins to be able to run `npm` builds and installs we must configure a `jenkins-build-slave` for Jenkins to use. This slave will be dynamically provisioned when we run a build. It needs to have NodeJS and npm installed in it. These slaves can take a time to build themselves so to speed up we have placed the slave within OpenShift and you can use the following commands to be able to use them in your project.
```

Comment:
I'd probably keep some of this to explain the "black magic" ... maybe just remove the last part (i.e.: "... and you can use the following commands ....") and replace it with something like "... and an ImageStream, with the "role=jenkins-slave" label, is added by the Configuration-as-Code run managed by the [openshift-applier](https://github.com/redhat-cop/openshift-applier) run below. "